doc/schema.html

```diff
@@ -3488,6 +3489,10 @@
       	<td><code>&lt;<a name="XML_AggFactCount" href="xml_schema.html#AggFactCount">AggFactCount</a>&gt;</code></td>
       	<td>Specifies name of the column in the candidate aggregate table which contains the number of fact table rows.</td>
     </tr>
+	<tr>
+		<td><code>&lt;<a name="XML_AggMeasureFactCount" href="xml_schema.html#AggMeasureFactCount">AggMeasureFactCount</a>&gt;</code></td>
+		<td>Specifies name of the column in the candidate aggregate table which contains the number of fact table rows associated with a specific measure.</td>
+	</tr>
```

Comment:
To clarify intent, maybe something like:  "the number of non-null fact table rows associated with a specific fact column.  Mondrian can use this information when calculating averages from aggregate table values."